src/corelibs/U2Formats/src/MSFFormat.cpp

```diff
@@ -160,64 +154,39 @@ void MSFFormat::load(IOAdapterReader &reader, const U2DbiRef &dbiRef, QList<GObj
         coreLog.info(tr("File check sum is incorrect: expected value: %1, current value %2").arg(checkSum).arg(sum));
     }
 
-    // read data
-    QRegExp coordsRegexp("^\\s+\\d+(\\s+\\d+)?\\s*$");
-    QRegExp onlySpacesRegexp("^\\s+$");
-
-    QMap<QString, int> processedDuplicatedNames;
-
+    // Read sequences.
+    QRegExp coordsRegexp("^\\d+(\\s+\\d+)?$");
```

Comment:
`(\\s+\\d+)?` - this means, that the second number either exists or not, but I haven't found an example of the "MSF" file where coordinate line has only one number. Maybe the following regexp will be a bit simpler : `^\\d+\\s+\\d+?$`

Reply:
We have an example like this, check test7.msf (check the very end of the file)
I do not know where it came from but it looks pretty real.